src/content/en/updates/2017/09/auto-play-policy-changes.md

```diff
@@ -0,0 +1,154 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Learn best practices for good user experiences with the new auto-play policies in Chrome.
+
+{# wf_updated_on: 2017-09-04 #}
+{# wf_published_on: 2017-09-04 #}
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Blink>Media #}
+
+# Auto-Play Policy Changes {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Chrome's auto-play policies are about to change in 2018 and I'm here to tell
+you why and how this is going to affect video playback with sound. Spoiler
+alert: Users are going to love it!
+
+<figure>
+  <img src="/web/updates/images/2017/09/autoplay-memes.png"
+       alt="Internet memes tagged "auto-play">
+  <figcaption>
+    <b>Figure 1.</b>
+    Internet memes tagged "auto-play"
+  </figcaption>
+</figure>
+
+## New Behaviors {: #new-behaviors }
+
+As you may have [noticed], web browsers are moving towards stricter auto-play
+policies in order to improve the web experience for users, minimize the
+incentives to install extensions that block ads, and reduce data consumption on
+expensive and/or constrained network.
+
+With these new auto-play policies, the Chrome team aims to provide a greater
+control to users over content playing in their browser. It will also benefit
+publishers who have legitimate auto-play use cases.
+
+Chrome's auto-play policies are simple:
+
+- Muted auto-play is always allowed.
+- Auto-play with sound is allowed if any of the following conditions are met:
+    - User interaction on origin (button click, document activation,
+      navigation, etc.)
+    - [Media Engagement Index](#mei) threshold is crossed (desktop only)
+    - User has added a [PWA] to their homescreen (mobile only)
+- Iframes will require auto-play [permission delegation](#iframe) from
```

Comment:
Not really require. Iframe might get user gesture. Otherwise, the top frame *can* delegate user gesture to their iframes.

Reply:
Done.